src/xiana/swagger.clj

```diff
@@ -0,0 +1,47 @@
+(ns xiana.swagger
+  (:require
+    [jsonista.core :as json]
+    [xiana.route.helpers :as helpers]))
+
+(defn routes->swagger-json [routes
+                            & {type :type
+                               render? :render?
+                               route-opt-map :route-opt-map}]
+  (-> routes
+      helpers/routes->routes'
+      ((if (not (false? render?))
+         #(helpers/routes->swagger-data % :route-opt-map route-opt-map)
+         identity))
+      ((cond
+         (= type :json) json/write-value-as-string
+         (= type :edn) identity
+         :else identity))))
+
+(defn swagger-configs-there?
+  "Checks if the config has the required keys for swagger functionality.
+   Required keys:
+   * :xiana/swagger
+   * :xiana/swagger-ui"
+  [config]
+  (->> config
+       ((juxt :xiana/swagger-ui :xiana/swagger))
+       (reduce #(and % %2) true)
+       boolean))
```

Comment:
(apply and ((juxt :xiana/swagger-ui :xiana/swagger) config))

or

(every? true? ((juxt :xiana/swagger-ui :xiana/swagger) config))

Reply:
`(apply and ((juxt :xiana/swagger-ui :xiana/swagger) config))` is the right reflection of that s-expression without overseeing any future changes. Fixed, thanks.